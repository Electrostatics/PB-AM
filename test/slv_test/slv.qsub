#!/bin/bash
#$ -cwd
#$ -o slv.out
#$ -j y

EXEC="../../bin/mpe"

$EXEC slv barnaseh.pqr 2 15.0  
$EXEC slv barnaseh.pqr 4 15.0  
$EXEC slv barnaseh.pqr 8 15.0  

