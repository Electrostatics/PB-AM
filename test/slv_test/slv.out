SOLVE
barnaseh.pqr: charges: 1703, net charge: 2, radius: 31.332, cen: [20.325,44.233,12.414]
Building system is complete
MOLECULE #1
	ENERGY: 0.0082238
	FORCE: 0.0010341 [0.00011834,0.00022371,-0.0010027]
	TORQUE: 0.0057479 [-0.0029764,0.0048316,0.00091393]
MOLECULE #2
	ENERGY: 0.0082238
	FORCE: 0.0010341 [-0.00011834,-0.00022371,0.0010026]
	TORQUE: 0.015364 [-0.014815,0.0039322,-0.0010559]
SOLVE
barnaseh.pqr: charges: 1703, net charge: 2, radius: 31.332, cen: [20.325,44.233,12.414]
saving files:
	barna_2P.mdp
	barna_2P_sp.mdp
Building system is complete
MOLECULE #1
	ENERGY: 0.0082238
	FORCE: 0.0010341 [0.00011834,0.00022371,-0.0010027]
	TORQUE: 0.0057479 [-0.0029764,0.0048316,0.00091393]
MOLECULE #2
	ENERGY: 0.0082238
	FORCE: 0.0010341 [-0.00011834,-0.00022371,0.0010026]
	TORQUE: 0.015364 [-0.014815,0.0039322,-0.0010559]
